src/visualize/q_sphere.py

```diff
@@ -1,8 +1,10 @@
 import matplotlib.pyplot as plt
 from numpy import pi, log2, ndarray, cos, sin, linspace
 import math
+import re
 from typing import Union
 from ..quantum_circuit import QuantumCircuit
+from ..errors import *
```

Comment:
IMO, we should avoid importing using `*`. It pollutes the namespace. I would import by doing
`from ..errors import InvalidSavePathError`

Reply:
Resolved in commit.